### PR TITLE
C100 dashboard tasklist changes

### PR DESCRIPTION
**Pull request by channa-patil**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -52,6 +51,7 @@
     <cve>CVE-2022-45143</cve>
     <cve>CVE-2022-42003</cve>
     <cve>CVE-2022-42004</cve>
+    <cve>CVE-2022-1471</cve>
```
Comment (AlokDatta) [inline config/owasp/suppressions.xml:54]: we dont need this. can we remove this please.

File: src/main/java/uk/gov/hmcts/reform/prl/constants/PrlAppsConstants.java [hunk 2]
```diff
@@ -340,5 +340,9 @@ public class PrlAppsConstants {
     public static final String CASE_NAME_HMCTS_INTERNAL = "caseNameHmctsInternal";
     //C100 default court details
     public static final String C100_DEFAULT_COURT_NAME = "STOKE ON TRENT TRIBUNAL HEARING CENTRE";
+    public static final String CLOSED_STATE = State.ALL_FINAL_ORDERS_ISSUED.getValue();
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/constants/PrlAppsConstants.java:343]: Can we use state.java please for this? (line 343)

File: src/main/java/uk/gov/hmcts/reform/prl/utils/CaseUtils.java [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
+import java.util.List;
 
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.CLOSED_STATE;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.DRAFT_STATE;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.GATEKEEPING_STATE;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.ISSUED_STATE;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.PENDING_STATE;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.RETURN_STATE;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.SUBMITTED_STATE;
+import static uk.gov.hmcts.reform.prl.constants.PrlAppsConstants.WITHDRAWN_STATE;
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/utils/CaseUtils.java:25]: We should refer to the Enum State.java

File: src/main/java/uk/gov/hmcts/reform/prl/constants/PrlAppsConstants.java (hunk 2 above)
Comment (channa-patil) [reply src/main/java/uk/gov/hmcts/reform/prl/constants/PrlAppsConstants.java:343]: Hi @AlokDatta, It's using State.java but these have defined as constants with meaningful state names.

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (channa-patil) [reply config/owasp/suppressions.xml:56]: Sure, build was failing bcoz this cve not added in suppression so I have added it.
I will remove it, thanks

**Review by yogendraU — APPROVED** [review]

**Pull request merged**